src/app/core/authentication/authentication.interceptor.ts

```diff
@@ -18,6 +18,7 @@ const httpOptions = {
 
 /** Authorization header. */
 const authorizationHeader = 'Authorization';
+const authorizationTenantHeader = 'fineract-platform-tenantid';
```

Comment:
I think it is `Fineract-Platform-TenantId`

Reply:
Fixed